firestore-bigquery-export/firestore-bigquery-change-tracker/src/logs.ts

```diff
@@ -165,6 +166,70 @@ export const timestampMissingValue = (fieldName: string) => {
   );
 };
 
-export const addDocumentIdColumn = (table) => {
-  logger.log(`Updated '${table}' table with a 'document_id' column`);
+export const addNewColumn = (table: string, field: string) => {
+  logger.log(`Updated '${table}' table with a '${field}' column`);
+};
+
+export const addPartitionFieldColumn = (table, field) => {
+  logger.log(
+    `Updated '${table}' table with a partition field '${field}' column`
+  );
+};
+
+export const firestoreTimePartitionFieldError = (
+  documentName: string | undefined,
+  fieldName: string | undefined,
+  firestoreFieldName: string | undefined,
+  firestoreFieldData: any
+) => {
+  logger.warn(
+    `Wrong type of Firestore Field for TimePartitioning. Accepts only strings in BigQuery format (DATE, DATETIME, TIMESTAMP) and Firestore Timestamp. Firestore Document field path: ${documentName}. Field name: ${firestoreFieldName}. Field data: ${firestoreFieldData}. Schema field "${fieldName}" value will be null.`
+  );
+};
+
+export const firestoreTimePartitioningParametersError = (
+  fieldName: string | undefined,
+  fieldType: string | undefined,
+  firestoreFieldName: string | undefined,
+  dataFirestoreField: firestore.Timestamp | string | undefined
+) => {
+  logger.warn(
```

Comment:
This should be a documentation message, should only be logging if this condition is not met?

Would also be `log.info` if it was general information.